packages/fxa-settings/src/models/hooks.ts

```diff
@@ -95,6 +97,40 @@ export function useRelier() {
   return relierFactory.getRelier();
 }
 
+export function useUrlSearchContext() {
```

Comment:
@LZoog Not user about these hooks. Maybe this should just be be pulled off app context directly?